### PR TITLE
[Engine][Example] Fix engine example can't run error

### PR DESCRIPTION
**Pull request by Hisoka-X**
## Purpose of this pull request
Fix engine example can't run error, because the Fake source config not update

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-examples/seatunnel-engine-examples/src/main/resources/examples/fake_to_console.conf [hunk 1]
```diff
@@ -30,14 +30,24 @@ source {
...
+    schema = {
+      fields {
+        name = "string"
+        age = "int"
+      }
+    }
   }
 
   FakeSource {
     result_table_name = "fake"
```
Comment (hailin0) [inline seatunnel-examples/seatunnel-engine-examples/src/main/resources/examples/fake_to_console.conf:43]: duplicate result_table_name?

Comment (liugddx) [reply seatunnel-examples/seatunnel-engine-examples/src/main/resources/examples/fake_to_console.conf:43]: > duplicate result_table_name?

parallelism of simulation?

Comment (Hisoka-X) [reply seatunnel-examples/seatunnel-engine-examples/src/main/resources/examples/fake_to_console.conf:43]: This is a feature that multi-source to one sink use same result_table_name.

**Review by EricJoy2048 — APPROVED** [review]
LGTM

**Review by TaoZex — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**